submissions/busolaAdedoyin/javascript/index.js

```diff
@@ -0,0 +1,27 @@
+
+var x = "";
+var productList = [
+    Fabric1  = {
```

Comment:
there is no need for naming this object. Simply declare the object. Remove the Fabric1 same goes to others